index.html

```diff
@@ -0,0 +1,242 @@
+                 <h1>About </br>
+                     Myself</h1>
+                    
+                        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                        </div>        
+                    </div>
+        <button class="con">LET'S CONNECT </button>
+        <div class="social"> 
+            <ul class="head">    
+            <li class="twitter"><a href="#" class="scIcon"><img src="/img/social/Twitter (2).svg" alt=""></a></li>
+        
+            <li class="linkedin"><a href="#" class="scIcon"><img src="img/social/Linkedin icon (1).svg" alt=""></a></li>
+        
+            <li class="medium"><a href="#" class="scIcon"><img src="img/social/Medium (2).svg" alt=""></a></li>
+        
+            <li class="github"><a href="#" class="scIcon"><img src="img/social/gihub.svg" alt=""></a></li>
+        
+            <li class="angel"><a href="#" class="scIcon"><img src="img/social/angel.svg" alt=""></a></li>
+            </ul>
+         </div>  
+         <button class="bt">Get my resume</button>
+        </div>
+        <div class="lang">
+           <div class="languages">
+            <h2>Languages</h2>
+            <img src="/img/v.svg" alt="">
+            </div>
+           <div class="icons">
+            <div class="box">
+            <img src="img/Ellipse 1.svg" alt="">
+            <h2>Java Script</h2>
+            </div>
+    
+            <div class="box">
+                <img src="img/Ellipse 1 (1).svg" alt="">
+                <h2>HTML</h2>
+                </div>
+    
+            <div class="box">
+                 <img src="img/Ellipse 1 (2).svg" alt="">
+                 <h2>CSS</h2>
+            </div>    
+    
+           </div> 
+           <hr>
+           <div class="titleArrow">
+            <p>Frameworks</p>
+            <img src="img/Union.svg">
+        </div>
+        <hr>
+        <div class="titleArrow">
+            <p>Skills</p>
+            <img src="img/Union.svg">
+        </div>
+        <hr>
+        </div>
+    
+    </section>
+    <section class="myForm">
+        <div class="containerForm">
+        <h2 class="contact-title">Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xzbqrjrj" method="post">
+          <div class="contactmeul">
+           <input class="input" type="text" maxlength="30" placeholder="Name" required> 
+           <input class="input" type="email" placeholder="E-mail" required>
+           <textarea class="textarea" name="message" maxlength="500" placeholder="write your message here" required></textarea>
+           </div>
+           <button type="submit">Get in touch</button>
```

Comment:
The button "Let's Connect" could be improved by making it more descriptive, such as "Contact Me" or "Get in Touch".

Reply:
- [ ] Nice job👍 with the improvement suggestion, but you should also provide the 'why'🤷‍♂️ behind your suggestion. This will help the person 👩‍🦳you give feedback to understand and make sense of your suggestions

- [OPTIONAL] Please consider using emojis👩‍🦰 in your comments, They make the code review more human-friendly😃